pallet/src/tests.rs

```diff
@@ -1,7 +1,57 @@
 #![cfg(test)]
 
-use crate::mock::*;
+use crate::{migrations, mock::*, pallet};
 use frame_support::assert_ok;
+use frame_support::traits::{GetStorageVersion, StorageVersion};
+
+#[test]
```

Comment:
should we also test that nothing is done when the storage version is already updated?

Reply:
done